File: server/models/Article.js
const mongoose = require('mongoose')

const articleSchema = new mongoose.Schema({
    title: {
        type: String,
        required: true,
        default: "Title unavailable."
    },
    author: {
        type: String,
        required: true,
        default: "Author date unavailable."
    },
    date: {
        type: String,
        required: true,
        default: "Publish date unavailable."
    },
    summary: {
        type: String,
        required: true,
        default: "Summary unavailable."
    },
    url: {
        type: String,
        required: true,
        default: "URL unavailable." 
    }
})

module.exports = mongoose.model('Article', articleSchema)

File: server/services/webscraper/pyscraper/models/Article.py
from mongoengine import *

class Article(Document):
    title = StringField(required=True, default="Title unavailable.")
    author = StringField(required=True, default="Author date unavailable.")
    date = StringField(required=True, default="Publish date unavailable.")
    summary = StringField(required=True, default="Summary unavailable.")
    url = StringField(required=True, default="URL unavailable.")

    meta = {
        'collection': 'articles',  # Specifies the collection name. Default would have been 'article'
        'strict': False # allows validation to skip extra fields that may be present in exisiting documents created by JS
    }